json-rpc/types/Quantity.md

```diff
@@ -0,0 +1,33 @@
+---
+title: JSPON RPC Quantity Type Spec
+author: Alita Moore (@alita-moore)
+discussions-to: https://github.com/ethereum-oasis/eth1.x-JSON-RPC-API-standard
+created: 2021-04-13
+---
```

Comment:
I don't think metadata is required.  We can get most of it from other places like GitHub history, file name, etc. and I think the plan is to have discussion just take place on the PR directly.